processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -831,7 +831,7 @@ public String apply(String dimValue)
         .build();
 
     List<Row> expectedResults = Arrays.asList(
-        GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-01", "alias", "", "rows", 3L, "idx", 2870L),
+        GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-01", "alias", null, "rows", 3L, "idx", 2870L),
```

Comment:
IncrementalIndex regards null and empty string altogether as empty string and stores them like that in fact map. But when retrieving values, which uses dimdim.getValue(), it converts empty strings to nulls. So we can say most of queries returns null for empty string. 

But for group-by queries, it makes IncrementalIndex in the end of query and retrieves values from the fact map in it directly(in `IncrementablIndex#iterableWithPostAggregations`), which makes empty strings returned as-is.

With this patch, there is no direct way to retrieve values from fact map because now we stores int[] in it rather than string[]. So I can say that it's more consistent than before. We can simply revert nulls to empty strings again in the method to make the test pass but I think it's not a good idea. Could you make decision on this, @fjy?